using-timescaledb/distributed-hypertables.md

```diff
@@ -0,0 +1,124 @@
+symbol, etc.) so that an individual time interval is spread over all data nodes
+(although this is not required).  Thus to create a distribted hypertable, you
+need to have added at least one data node before starting.
+
+Data nodes together with an *access node* constitute the
+multi-node TimescaleDB architecture
+([architecture][]). All the nodes are TimescaleDB instances,
+i.e., hosts with a running PostgreSQL server and loaded TimescaleDB extension.
+While the data nodes store distributed chunks, the access node is
+the entry point for clients to access distributed hypertables.
+
+For more information about setting up a multi-node architecture, including
+configuring access nodes and data nodes, please follow our [getting started
+with multi-node instructions][getting-started-multi-node].
+
+>:WARNING: There are some [limitations][distributed-hypertable-limitations] to
+using distributed hypertables that might be good to review before getting
+started.
+
+### Creating Distributed Hypertables [](create)
+
+As with normal hypertables, start by creating a regular
+table to hold the data, then transform it into a distributed
+hypertable. In contrast to creating a normal hypertable, we use
+the function
+[`create_distributed_hypertable`][create_distributed_hypertable].
+
+```sql
+CREATE TABLE conditions (
+  time        TIMESTAMPTZ       NOT NULL,
+  location    TEXT              NOT NULL,
+  temperature DOUBLE PRECISION  NULL,
+  humidity    DOUBLE PRECISION  NULL
+);
+
+SELECT create_distributed_hypertable('conditions', 'time', 'location');
+```
+
+>:WARNING: If there are no data nodes available, you will get an error
+>when executing `create_distributed_hypertable` and the distributed
+>hypertable will not be created.
+
+This creates a multi-dimensional distributed hypertable
+partitioned along `time` and `location`.
+You can now insert data into the distributed hypertable and
+it will automatically be partitioned on the available data nodes
+by using the provided space partition. You
+can find more information for how to work with data in hypertables in the
+section [Creating Hypertables][creating-hypertables].
+
+### Changing the Number of Data Nodes for a Distributed Hypertable [](changing-data-nodes)
+
+By default, distributed hypertables will use all available data nodes,
+but individual data nodes can be added and removed from already
+created distributed hypertables using
+[`add_data_node`][add_data_node]/[`attach_data_node`][attach_data_node] and
+[`detach_data_node`][detach_data_node].
+
+For example, if you add a new data node to the distributed database,
+it will not automatically be
+added to existing distributed hypertables, so it is necessary to attach
+it explicitly.
+
+```sql
+SELECT add_data_node('node3', host => 'dn3.example.com');
+
+SELECT attach_data_node('node3', hypertable => 'conditions');
+```
+
+Once the data node has been attached, any new chunks will also be
+distributed to the hypertable on the new data node.
+
```

Comment:
Seems repetitive, you can probably remove it.

Reply:
I think it adds some good information, so probably will leave.